src/scp/Slot.cpp

```diff
@@ -127,13 +128,14 @@ Slot::processEnvelope(SCPEnvelope const& envelope, bool self)
             res = mBallotProtocol.processEnvelope(envelope, self);
         }
     }
-    catch (...)
+    catch (std::exception& e)
     {
-        auto info = getJsonInfo();
-        CLOG(ERROR, "SCP") << "Exception in processEnvelope "
-                           << "state: " << info.toStyledString()
-                           << " processing envelope: "
-                           << mSCP.envToStr(envelope);
+        CLOG(FATAL, "SCP") << "SCP context:";
```

Comment:
I don't think you need this change: as we rethrow, the exception string should already be logged upstream

Reply:
Do you mean whole change or just adding e.what() to the message?

Reply:
this particular change moving `catch(...)` to `catch(std::exception)` in order to log `e.what()`